src/main/java/bg/startit/hackathon/airquiality/email/EmailService.java

```diff
@@ -0,0 +1,52 @@
+package bg.startit.hackathon.airquiality.email;
+
+import bg.startit.hackathon.airquiality.model.User;
+import sendinblue.ApiClient;
+import sendinblue.ApiException;
+import sendinblue.Configuration;
+import sendinblue.auth.ApiKeyAuth;
+import sibApi.SmtpApi;
+import sibModel.CreateSmtpEmail;
+import sibModel.SendSmtpEmail;
+import sibModel.SendSmtpEmailSender;
+import sibModel.SendSmtpEmailTo;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class EmailService
+{
+  public boolean send(User user) {
+    ApiClient defaultClient = Configuration.getDefaultApiClient();
+
+    // Configure API key authorization: api-key
+    ApiKeyAuth apiKey = (ApiKeyAuth) defaultClient.getAuthentication("api-key");
+    apiKey.setApiKey("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");
```

Comment:
This is a security issue. Let's discuss that together, and I'll let you know how to fix it.